fem/sidredatacollection.cpp

```diff
@@ -385,38 +385,41 @@ createMeshBlueprintTopologies(bool hasBP, const std::string& mesh_name)
                             ? mesh->GetNE()
                             : mesh->GetNBE();
 
-   MFEM_VERIFY(num_elements > 0,
-               "TODO: processors with 0 " << mesh_name << " elements");
-
-   const int element_size = !isBdry
-                            ? mesh->GetElement(0)->GetNVertices()
-                            : mesh->GetBdrElement(0)->GetNVertices();
-
-   const int num_indices = num_elements * element_size;
-
-   // Find the element shape
-   // Note: Assumes homogeneous elements, so only check the first element
-   const int geom =
-      isBdry ?
-      mesh->GetBdrElementBaseGeometry(0) :
-      mesh->GetElementBaseGeometry(0);
-   const std::string eltTypeStr =
-      !isBdry
-      ? getElementName( static_cast<Element::Type>(
-                           mesh->GetElement(0)->GetType() ) )
-      : getElementName( static_cast<Element::Type>(
-                           mesh->GetBdrElement(0)->GetType() ) );
-
    const std::string mesh_topo_str = "topologies/" + mesh_name;
    const std::string mesh_attr_str = mesh_name + "_material_attribute";
 
+   int element_size = 0;
+   int num_indices = 0;
+   int geom = 0;
+   std::string eltTypeStr = "point";
```

Comment:
The blueprint for the ranks with no boundary elements will indicate that their 0-element boundary meshes are composed of "points" since didn't seem worthwhile to MPI reduce the element type.
I can add this, if desired.